diff --git a/websocket/client.go b/websocket/client.go
index 1a56cc1..3044515 100644
--- a/websocket/client.go
+++ b/websocket/client.go
@@ -40,7 +40,7 @@ const (
 	pongWait = 60 * time.Second
 
 	// Send pings to peer with this period. Must be less than pongWait.
-	pingPeriod = (pongWait * 9) / 10
+	pingPeriod = (pongWait * 5) / 10
 
 	// Maximum message size allowed from peer.
 	maxMessageSize = 1048576 // 1 MB
diff --git a/workers/websocket/agent/agent.go b/workers/websocket/agent/agent.go
index 262bcc4..4eeda0d 100644
--- a/workers/websocket/agent/agent.go
+++ b/workers/websocket/agent/agent.go
@@ -32,7 +32,7 @@ const (
 	pongWait = 60 * time.Second
 
 	// Send pings to peer with this period. Must be less than pongWait.
-	pingPeriod = (pongWait * 9) / 10
+	pingPeriod = (pongWait * 5) / 10
 
 	// Maximum message size allowed from peer.
 	maxMessageSize = 1048576 // 1 MB
